docs/release-notes/2.11.0.md

```diff
@@ -24,25 +24,30 @@ performance.
 
 ## New Features
 
-* [PMM-6567](https://jira.percona.com/browse/PMM-6567): Management API for new pg_stat_monitor PostgreSQL extension
+* [PMM-6567](https://jira.percona.com/browse/PMM-6567): New PostgreSQL extension 'pg_stat_monitor'
 * [PMM-6515](https://jira.percona.com/browse/PMM-6515): Link added directly to Node/Service page from Query Analytics filters, opens in new window
-* [PMM-6463](https://jira.percona.com/browse/PMM-6463): User interface and CLI changes for new pg_stat_monitor PostgreSQL extension
 
 
 
 ## Improvements
 
-* [PMM-6727](https://jira.percona.com/browse/PMM-6727): Update Grafana plugin grafana-polystat-panel to latest v1.2.2
+* [PMM-6727](https://jira.percona.com/browse/PMM-6727): Grafana plugin updates: grafana-polystat-panel=1.2.2, grafana-piechart-panel=1.6.1
 * [PMM-6625](https://jira.percona.com/browse/PMM-6625): Default sort to "Average - descending" on all dashboards
 * [PMM-6609](https://jira.percona.com/browse/PMM-6609): MySQL Instances Compare & Summary dashboards: Changed metric in 'MySQL Internal Memory Overview'
+* [PMM-6598](https://jira.percona.com/browse/PMM-6598): Improved wording on Dashboard image 'Share Panel'
```

Comment:
Improved wording and link to instructions on how to configure image sharing added

Reply:
Edited Jira summary:
"Dashboard image sharing (Share Panel): Improved wording with link to configuration instructions"